app/src/main/java/com/itsaky/androidide/viewmodel/EditorViewModel.kt

```diff
@@ -259,6 +293,14 @@ class EditorViewModel : ViewModel() {
     }
   }
 
+  fun handleWorkspaceSettingsJobCompletion(it: Job, operation: String) {
+    it.invokeOnCompletion { err ->
+      if (err != null) {
+        ILogger.ROOT.error("[EditorViewModel] Failed to {} workspace settings", operation, err)
+      }
+    }
+  }

```

Comment:
> Instead of defining a new function, which just prints a different message, why not define a generic function to handle both cases (`handleOpenedFilesCacheJobCompletion` and `handleWorkspaceSettingsJobCompletion`)?

I'll fix this.